src/vcpkg/sourceparagraph.cpp

```diff
@@ -1616,11 +1613,12 @@ namespace vcpkg
             auto maybe_configuration = reader.visit(*configuration, get_configuration_deserializer());
             if (!reader.errors().empty())
             {
-                print2(Color::error, "Errors occurred while parsing ", ManifestDeserializer::VCPKG_CONFIGURATION, "\n");
+                msg::println_error(msgErrorWhileParsing, msg::path = ManifestDeserializer::VCPKG_CONFIGURATION);
```

Comment:
This is wrong, it should be listing the file on each line with an error, but it faithfully preserves the previous behavior. It does add "error: " in front of everything but I think that's OK. (Here's the old behavior:)

Reply:
![image](https://user-images.githubusercontent.com/1544943/180311726-3328bb30-9684-4e60-8172-efdd3c793748.png)